game.cc

```diff
@@ -1,49 +1,606 @@
 #include "game.h"
+#include "piece.h"
+#include <iostream>
+
+using namespace std;
+/*
+TO DO:
+- En Passant
+- setup() function
+- clearGame()
+*/
+
+
 
 Game::Game(){
+	//Sets all pieces in Board to NULL
+	for(int i = 0; i < 8; i++){
+		for (int j = 0; j < 8; j++){
+			theBoard[i][j] = NULL;
+		}
+	}
 
+	for (int i = 0; i < 25; i++){
+		playerWhite[i] = NULL;
+		playerBlack[i] = NULL;
+	}
+	
+	p1Score = 0;
+	p2Score = 0;
+	currentPlayer = WHITE;
 }
 
 Game::~Game(){
+	clearGame();
+}
 
+void Game::notify(int x, int y, char c){
+	notifications->notify(x,y,c);
 }
 
 void Game::clearGame(){
-
+	delete [] playerWhite;
+	delete [] playerBlack;
+	delete [] theBoard;
```

Comment:
Since theBoard is a two-dimensional array, I believe you need to use a for loop to delete each row (which is a Piece* array) and then delete [] theBoard (which is a Piece** array).